core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -155,7 +155,7 @@ class TagsBlockingFirstPaint extends FRGatherer {
    */
   static async findBlockingTags(driver, networkRecords) {
     const firstRequestEndTime = networkRecords.reduce(
-      (min, record) => Math.min(min, record.endTime),
+      (min, record) => Math.min(min, record.networkEndTime),
```

Comment:
Would it make sense to use `mainThreadEndTime` here?

Reply:
Hmm... probably! I suppose we would be missing out on JS execution time here for scripts, which would be a big oversight.

Reply:
note: need to change down below too